x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -130,6 +130,15 @@ Service.
 `manage_transform`::
 All operations related to managing {transforms}.
 
+`manage_autoscaling`::
+All deployment operations, create or update autoscaling policy API (Need to be reviewed).
+
+`manage_data_frame_transforms`::
+Enable you to manage transforms (Need to be reviewed).
```

Comment:
```suggestion
All operations related to managing {transforms}.
```
This privilege is deprecated since 7.5. It grants exactly the same permissions as `manage_transform`.